### PR TITLE
New Keyboard: Southpaw75

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

New keyboard from HolySwitch Co which is a 60% with a numpad on the left hand side. 

Original QMK firmware code submitted by drewguy aka campingcampfire. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/holyswitch/southpaw75/config.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x484F // "HO" short for Holy Switch
+#define PRODUCT_ID      0x5350 // "SP" short for Southpaw
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    drewguy
+#define PRODUCT         southpaw default
+#define DESCRIPTION     QMK Firmware: southpaw default
```
Comment (tzarc) [inline keyboards/holyswitch/southpaw75/config.h:28]: remove this as per PR checklist

**Comment by fauxpark** [comment]
Dupe of https://github.com/qmk/qmk_firmware/pull/9844 ?

File: keyboards/holyswitch/southpaw75/config.h (hunk 1 above)
Comment (mechmerlin) [reply keyboards/holyswitch/southpaw75/config.h]: Done

**Comment by mechmerlin** [comment]
> Dupe of #9844 ?

I think it's just a similarly named board. This is the Southpaw done in collaboration with Drew Guy and HolySwitch Co.

**Comment by fauxpark** [comment]
The image in the readme is the same as that on the Holyswitch site :/
https://github.com/qmk/qmk_firmware/pull/9844/files#diff-deb12e387ce89cb4cba5b19501fc3fc4

**Comment by fauxpark** [comment]
Seems like there are two PCBs at play here, one which takes a Pro Micro-compatible board, and this one that I guess has an onboard 32U4. These should probably be combined into separate revisions of the same board.

**Comment by mechmerlin** [comment]
> Seems like there are two PCBs at play here, one which takes a Pro Micro-compatible board, and this one that I guess has an onboard 32U4. These should probably be combined into separate revisions of the same board.

This is very interesting, I have not been informed of this by the vendor, I'll reach out and have them clarify

**Comment by mechmerlin** [comment]
Alright I reached out to DrewGuy the designer of this PCB. 

This Southpaw75 with USB C is going to be sold **exclusively in the United States** and is being run by HolySwitch Co. The other Southpaw75 in PR #9844 is designed by Volvo and is going to be sold internationally through other vendors. 

With that information, I think this PR is still valid. I will however append the README with this information. This one can be contained within the HolySwitch directory and the other southpaw75 can be contained within the generic keyboards directory. Any thoughts/opinions?

**Comment by kelvinhall05** [comment]
Hi, figured I should clear something up :)

I created the Soutpaw75 and got Holyswitch as a US vendor. Drew was hired by Holyswitch to design a USB-C PCB with an integrated controller. While both this PCB and the original Pro Micro one should be compatible with the same QMK config, the matrix/controller pins are different so they're not compatible with the same firmware. I am in the process of adding the Pro Micro one to the configurator, and while that PR was still being reviewed, you added this PR for the Holyswitch PCB.

So, tl;dr, there are two different PCBs and both require their own QMK configs, and two separate PRs are open for each one.

Hopefully this clears things up lol. I do like the idea of having the USB-C one in a Holyswitch folder and mine in the generic keyboard directory.

**Comment by noroadsleft** [comment]
@kelvinhall05 Thanks for clearing this up. I actually just merged your PR for the Pro Micro version about ten minutes ago, and I was _quite_ confused when I saw the notification for this pull request.

**Comment by mechmerlin** [comment]
I've added the necessary info the readme

File: keyboards/holyswitch/southpaw75/info.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
+{
+    "keyboard_name": "Southpaw75",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 19,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 78,
```
Comment (noroadsleft) [inline keyboards/holyswitch/southpaw75/info.json:9]:
```suggestion
```

File: keyboards/holyswitch/southpaw75/info.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                {"label":"K80 (D5,B6)", "x":6.5, "y":4, "w":1.25},
+                {"label":"K81 (D5,F7)", "x":7.75, "y":4, "w":6.25},
+                {"label":"K82 (D5,F6)", "x":14, "y":4, "w":1.25},
+                {"label":"K83 (D5,F5)", "x":15.25, "y":4, "w":1.25},
+                {"label":"K84 (D5,F4)", "x":16.5, "y":4, "w":1.25},
+                {"label":"K85 (D5,D0)", "x":17.75, "y":4, "w":1.25}
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```
Comment (noroadsleft) [inline keyboards/holyswitch/southpaw75/info.json:92]:
```suggestion
```

File: keyboards/holyswitch/southpaw75/southpaw75.h [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    { K30,   K31,   K32,   K33,   K34,   K35,   K36,   K37,   K38   }, \
+    { K40,   K41,   K42,   K43,   K44,   K45,   K46,   K47,   K48   }, \
+    { K50,   K51,   K52,   K53,   K54,   K55,   K56,   K57,   K58   }, \
+    { K60,   K61,   K62,   K63,   K64,   K65,   K66,   K67,   K68   }, \
+    { K70,   K71,   K72,   K73,   K74,   K75,   K76,   K77,   K78   }, \
+    { K80,   K81,   K82,   K83,   K84,   K85,   KC_NO, KC_NO, KC_NO }, \
+}
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (noroadsleft) [inline keyboards/holyswitch/southpaw75/southpaw75.h:39]:
```suggestion
```

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!
